go-controller/pkg/ovn/controller/services/utils.go.old

```diff
@@ -0,0 +1,354 @@
+package services
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovn/gateway"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovn/loadbalancer"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog/v2"
+)
+
+func deleteVIPsFromOVN(vips sets.String, st *serviceTracker, name, namespace string) error {
```

Comment:
looks like you accidentally added this file "go-controller/pkg/ovn/controller/services/utils.go.old "

Reply:
Oops, removed it